app/controllers/admin_controller.rb

```diff
@@ -1,6 +1,11 @@
 class AdminController < ApplicationController
   
   def matched
+    matchings = Matching.all.includes(:student, :teacher)
+    @triplesArray = []
+    for matching in matchings do
+      @triplesArray.push({"matching": matching, "teacher": matching.teacher, "student": matching.student})
+    end
```

Comment:
can you call this matching_info instead of triplesArray? rails is usually snake case :)